Makefile

```diff
@@ -538,18 +538,6 @@ ifeq ($(AUTODESPAWN), true)
 	@scripts/setup_integration_env.sh pulsar stop
 endif
 
-.PHONY: test-integration-redis
-test-integration-redis: ## Runs Redis integration tests
-ifeq ($(AUTOSPAWN), true)
-	@scripts/setup_integration_env.sh redis stop
-	@scripts/setup_integration_env.sh redis start
-	sleep 10 # Many services are very slow... Give them a sec..
-endif
-	${MAYBE_ENVIRONMENT_EXEC} cargo test --no-fail-fast --no-default-features --features redis-integration-tests --lib ::redis:: -- --nocapture
-ifeq ($(AUTODESPAWN), true)
-	@scripts/setup_integration_env.sh redis stop
-endif
```

Comment:
What would you think of keeping this `make` target but having it run the commands you added directly to the workflow file for CI? I think this would make it easier to kick them off locally. It would also let you just keep the existing Github workflow in CI rather than adding a new job.

Reply:
About keeping the existing workflow, having this in the Makefile doesn't allow to stop the running containers if the job fails. We need to have this in a separate step to avoid letting hanging containers.

Reply:
Also, we can have a clear view of what has been migrated and what has not.